src/components/SigningModal/TransactionStepper/TransactionStepper.js

```diff
@@ -218,6 +249,29 @@ function TransactionStepper({ steps, onComplete, className }) {
           {layout === 'expanded' && renderSteps()}
         </ul>
       </div>
+      {infoMessage && (
+        <Info
+          css={`
+            margin-top: ${4 * GU}px;
+          `}
+        >
+          {' '}
+          {infoMessage}
+        </Info>
+      )}
+
+      {stepperStatus === STEPPER_ERROR && (
+        <Button
+          mode="strong"
+          wide
+          onMouseDown={handleSign}
```

Comment:
This is `onMouseDown` for now because of the event issues outlined here https://github.com/aragon/aragon-ui/issues/790